packages/xo-web/src/xo-app/user/index.js

```diff
@@ -282,6 +289,85 @@ const SshKeys = addSubscriptions({
   )
 })
 
+// ===================================================================
+const dateFormat = d3.timeFormat('%Y-%m-%d %H:%M')
+
+const COLUMNS_AUTH_TOKENS = [
+  {
+    itemRenderer: ({ id }) => (
+      <Copiable tagName='pre' data={id}>
+        {id.slice(4, 9)}
```

Comment:
Show the first chars instead.

This is not the case in backup code because older jobs had their identifier with a specific format: `id:<id>`.